crates/sui-json-rpc/src/object_changes.rs

```diff
@@ -8,9 +8,11 @@ use sui_types::base_types::{ObjectID, ObjectRef, SequenceNumber, SuiAddress};
 use sui_types::effects::ObjectRemoveKind;
 use sui_types::object::Owner;
 use sui_types::storage::WriteKind;
+use tracing::instrument;
 
 use crate::ObjectProvider;
 
+#[instrument(skip_all)]
```

Comment:
i think it could be useful to include a primary key in the attrs if there is an obvious candidate. For any method that takes a `TransactionDigest` that is probably the right one to use. For this one, maybe `sender`, but its up to you